kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+
+import sys
+from six.moves.xmlrpc_client import Fault
```

Comment:
You can use `pyupgrade` to remove Python 2 constructs because `kobo` no longer supports it: https://github.com/release-engineering/kobo/commit/065420cea72303d92656e1c1b67b2ca9f6e30c16